docs/SMCWG-charter.md

```diff
@@ -0,0 +1,334 @@
+services for electronic messaging applications, namely sender authentication,
+message integrity, and message privacy through encryption. For effective
+authentication and privacy, it is imperative that the CA validates the subject’s
+identity (if present) and its email address. The recipient of a digitally signed
+message can authenticate an email message to receive protection against email
+spoofing and can encrypt a response to the original sender by referring
+to the public key, email address, and distinguished name (if present) contained
+in an S/MIME certificate.
+
+The primary use case under consideration for the working group is a model
+whereby senders and recipients of email messages receive “reasonable assurance”
+that the other party to the communication identified in the certificate has
+control of the domain or email address being asserted. A variation of this
+primary use case is where an individual or organization digitally signs email to
+establish its authenticity and source of origin.
+
+Therefore, in order to provide reasonable assurance, it is crucial to establish
+a standard method to validate an email address and the subject’s identity (if
+present) prior to binding them to the public key. “Reasonable
+assurance” is to be determined and defined by this SMCWG through studying the
+existing methods that exist in the industry, as well as identity management
+frameworks and any applicable legislation.
+
+An S/MIME certificate can also be used in an automated message with transfer
+agents that use cryptographic security services that do not require any human
+intervention, such as the signing of software-generated documents and the
+encryption of fax messages sent over the Internet. While these existing use
+cases are not in scope for the SMCWG, care will be exercised by the SMCWG to
+avoid unintended adverse effects to these uses. The security, stability and
+resiliency of the Internet shall be taken into consideration when the SMCWG
+forms its consensus. The SMCWG will consult other technical communities when and
+as necessary.
+
+The problem to be addressed by the working group is the absence of consistent
+and audited validation practices used by CAs in establishing the identity of the
+subject (if present) and verifying that the subscriber controls the email
+address. While there are methods for validating control of a domain that can be
+adopted from TLS certificates, there are currently no standard requirements for
+validating control of email addresses. Methods for validating identities in TLS
+certificates also exist and should be leveraged where possible, as well as other
+identity validation standards common in the industry.
+At least one method to efficiently validate an email address as well as
+developing a consistent profile for S/MIME certificates in order to facilitate
+technical interoperability across the Internet are also to be addressed by
+the SMCWG.
+
```

Comment:
@timfromdigicert Does it make sense to bring the paragraph you mentioned in https://github.com/cabforum/documents/pull/167/files#r421068417 here?

I could also see it moved to Line 99 with some tweaks, if you want it more explicitly in scope.

Independent of the discussion on "out of scope", it does seem useful.

Reply:
I would support that.  I would put it at a "nice to have" level of importance.

Reply:
https://github.com/sleevi/cabforum-docs/pull/19